src/MPSKit.jl

```diff
@@ -27,7 +27,9 @@ export entanglementplot, transferplot
 
 # hamiltonian things
 export Cache
-export SparseMPO, MPOHamiltonian, DenseMPO, MPOMultiline, LazySum
+export SparseMPO, MPOHamiltonian, DenseMPO, MPOMultiline
+export UntimedOperator, TimedOperator, LazySum
```

Comment:
Do we need to export timed and untimed separately? Can we just expose `MultipliedOperator`?

Reply:
Yes we could. At the moment I've only written code for `TimedOperator` and `UntimedOperator` so the user could try to put something in MultipliedOperator which is not (yet) supported and encounter errors down the line. That is why only `TimedOperator` and `UntimedOperator` are exported.